syntax = "proto3";

option cc_enable_arenas = true;

package tensorflow.serving;

message BaseResponse {
    //for cms, check new model load valid
    string code = 1;
    string msg = 2;
}